.github/workflows/cicd-staging.yaml

```diff
@@ -0,0 +1,72 @@
+name: CI/CD Production
```

Comment:
Can we stick to just produciton for now?

Reply:
Sure, but we should probably keep "Production" in the name anyway?

Reply:
Yes this was in reference to this specific file existing `cicd-staging.yaml`